### PR TITLE
Tiny fixes

### PR DESCRIPTION
**Pull request by kendallstrautman**
Please pull these awesome changes in!

Just a few formatting changes and fixed a dead link.

### CODE REVIEW

File: content/docs/index.md [hunk 1]
```diff
@@ -1,64 +1,63 @@
 ---
-title: TinaCMS Documentation
+title: Tina is Not a CMS
```
Comment (kendallstrautman) [inline content/docs/index.md:2]: TinaCMS Documentation

Comment (kendallstrautman) [inline content/docs/index.md:2]:
```suggestion
title: TinaCMS Documentation
```

Comment (kendallstrautman) [reply content/docs/index.md:2]: looks like this wasn't updated from master or something?

File: content/docs/plugins/forms.md [hunk 2]
```diff
@@ -7,22 +7,22 @@ consumes:
...
 >
 > This document explains how to set up forms in any React project. If you're using Gatsby or Next.js, we have helper packages that streamline this process for specific workflows:
 >
-> - [Editing Markdown Files with Gatsby](/guides/gatsby/git/create-remark-form)
-> - [Editing JSON Files with Gatsby](/guides/gatsby/git/create-json-form)
-> - [Editing local Markdown Files via Git with Next.js](/guides/nextjs/git/using-markdown)
-> - [Editing local JSON Files via Git with Next.js](/guides/nextjs/git/creating-git-forms)
+> * [Editing Markdown Files with Gatsby](/guides/gatsby/git/create-remark-form)
+> * [Editing JSON Files with Gatsby](/guides/gatsby/git/create-json-form)
+> * [Editing local Markdown Files via Git with Next.js](https://tinacms.org/packages/next-tinacms-markdown)
```
Comment (ncphillips) [inline content/docs/plugins/forms.md:24]: So it works on locally too
```suggestion
> * [Editing local Markdown Files via Git with Next.js](/packages/next-tinacms-markdown)
```

**Pull request merged**